src/main/java/com/google/flourbot/entity/EntityModuleImplementation.java

```diff
@@ -91,7 +92,23 @@ public EntityModuleImplementation(DataStorage dataStorage) {
         String sheetAction = (String) actionData.get("sheetAction");
```

Comment:
Should convert to `ActionType` ENUM using `ActionType.valueOf(sheetAction);`

Reply:
Are we changing the representation in the DB? Right now it's still "Sheet Append Action" but if we change it to "SHEET_APPEND_ROW" then I think the casting is a great idea! Saves us the extra code

Reply:
Oh good point! The firestore representation is not all caps.